pages/api/models/ServicePackage.ts

```diff
@@ -0,0 +1,45 @@
+module.exports = (sequelize: any) => {
+
+    class ServicePackage extends Model {}
```

Comment:
Import missing

Reply:
If I do the import I get the following error
![image](https://user-images.githubusercontent.com/86666889/180807856-5e8018a1-2b55-4e3d-bfdd-5abf888acdb7.png)